bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -259,7 +269,7 @@ public Row readModifyWriteRow(ReadModifyWriteRowRequest request) {
   @Override
   public ListenableFuture<List<SampleRowKeysResponse>> sampleRowKeysAsync(
       SampleRowKeysRequest request) {
-    return BigtableAsyncUtilities.doReadAsync(retryOptions, request, sampleRowKeysAsync,
+    return BigtableAsyncUtilities.performAsyncRpc(retryOptions, request, sampleRowKeysAsync,
```

Comment:
I'm having some troubles understanding why do we resuse the sampleRowKeysAsync RetryableRpc, but create new ones for mutations.

Could this be made to match, say, checkAndMutateRow?